routes/apiRoutes.js

```diff
@@ -1,149 +1,81 @@
-/* eslint-disable no-console */
 import express from 'express';
 import sequelize from 'sequelize';
-
 import db from '../database/initializeDB.js';
+import date from '../controllers/date.js';
 
-const router = express.Router();
-
-/* Magnitude endpoint */
-router.route('/magnitude')
-  .get(async (req, res) => {
-    try {
-      console.log('touched /magnitude with GET');
-      res.json({ message: 'GET endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .put(async (req, res) => {
-    try {
-      console.log('touched /magnitude with PUT');
-      res.json({ message: 'PUT endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .post(async (req, res) => {
-    try {
-      console.log('touched /magnitude with POST');
-      res.json({ message: 'POST endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  })
-
-  .delete(async (req, res) => {
-    try {
-      console.log('touched /magnitude with DELETE');
-      res.json({ message: 'DELETE endpoint' });
-    } catch (err) {
-      console.log(err);
-    }
-  });
+/* eslint-disable no-console */
 
-router.get('/', (req, res) => {
-  res.send('Welcome to the West Coast Hurricane API!');
-});
+const router = express.Router();
 
+/* Date Endpoint */
 
-router.get('/hurricanes', async (req, res) => {
+router.get('/date', async(req,res) => {
   try {
-    const hurricanes = await db.hurricane.findAll();
-    const reply = hurricanes.length > 0 ? { data: hurricanes } : { message: 'no results found' };
-    res.json(reply);
+    console.log('touched /date with GET');
+    const result = await db.sequelizeDB.query(date.getDate, {
+      type:sequelize.QueryTypes.SELECT
+    });
+    res.json(result);
   } catch (err) {
-    console.error(err);
-    res.error('Server error');
+      console.log(error);
```

Comment:
Gotta have a response inside the catch or your server will break - `res.send({message: "something went wrong"})`  or similar. `res.error` is undefined though this is my bad